components/_patterns/01-atoms/04-images/icons/_icon.twig

```diff
@@ -13,6 +16,14 @@
 {% set icon_url = icon_url|default('/themes/custom/emulsify/dist/img/sprite/') %}
 {{ attach_library('emulsify/sprite') }}
 
-<svg {{ bem(icon_base_class, (icon_modifiers), icon_blockname, (icon_js_class)) }}>
+<svg {{ bem(icon_base_class, (icon_modifiers), icon_blockname, (icon_js_class)) }}
+  {% if icon_role %}
```

Comment:
I think this should check that ALL variables are included. We don't want to accidentally produce something like this if they only provide the role

`aria-labelledby="title-" aria-describedby="desc-" role="icon-role"`

We could get more granular with the `if` statements, but I'm fine with a "provide it all, or provide nothing` approach, and fallback to `aria-hidden` if they don't provide anything

Reply:
We could use `set` for each attribute on top and then each attribute_placeholder will either add (nothing) or will add the accurate attribute itself.